README.md

```diff
@@ -22,7 +23,21 @@ that is used for the implementation of the Northbound APIs.
 
 ## <a name="installation"/> Installation
 There are three ways of installing the Ultralight 2.0 Agent: cloning the Github repository, using the RPM or using Docker.
-The following sections describe each approach in detail.
+Regardless of the installation method, there are some middlewares that must be present, as a prerequisite for the component
+installation (no installation instructions are provided for these middlewares):
+
+* A MQTT Broker is needed for the MQTT Binding to work. Both [Mosquitto](http://mosquitto.org/) and [Rabbit MQ](https://www.rabbitmq.com/)
```

Comment:
Version information should be included. E.g. in the case of MongoDB, required version should be 3.2 (according to the general requirement we have in IoTP).